docs/blueprints/deletion-performance.md

```diff
@@ -0,0 +1,162 @@
+</tbody>
+</table>
+
+We extracted the data of the table the following trace graph files:
+
+* [1kib](images/deletion-perfomance-1.0kb-file-trace.svg)
+* [4Kib](images/deletion-perfomance-4.0kb-file-trace.svg)
+* [5Kib](images/deletion-perfomance-5.0kb-file-trace.svg)
+* [10kib](images/deletion-perfomance-10.0kb-file-trace.svg)
+* [2.2Mib](images/deletion-perfomance-2.2mb-file-trace.svg)
+* [256Mib](images/deletion-perfomance-256mb-file-trace.svg)
+
+
+## Design
+
+Delegate to the satellite to make the delete requests to the storage nodes.
+
+Uplink will send the delete request to the satellite and wait for its response.
+
+The satellite will send the delete requests to all the storage nodes that have segments and delete the segments from the pointerDB. Finally, it will respond to the uplink.
+
+The satellite will communicate with the storage nodes:
+
+- Using the current network protocol, RPC/TCP.
+- The request will be sent with long-time cancellation in the same way as upload.
```

Comment:
Perhaps you meant "long-tail cancellation"?

Reply:
yes, I think that auto corrector played bad and I didn't realized.